core/src/main/java/io/questdb/cairo/wal/WalEventCursor.java

```diff
@@ -326,6 +348,47 @@ private void read() {
         }
     }
 
+    public class DataInfoExt extends DataInfo {
+        private long lastRefreshBaseTableTxn;
+        private long lastRefreshTimestamp;
+
+        public long getLastRefreshBaseTableTxn() {
+            return lastRefreshBaseTableTxn;
+        }
+
+        public long getLastRefreshTimestamp() {
+            return lastRefreshTimestamp;
+        }
+
+        @Override
+        protected void read() {
+            super.read();
+            // read the extra fields in the fixed part
+            // symbol map will start after this
+            lastRefreshBaseTableTxn = readLong();
+            lastRefreshTimestamp = readLong();
```

Comment:
This is the incompatible format, older version will not be able to read it.
Please bump `WalUtils.WAL_FORMAT_VERSION` and test that this new code can read both version 0 and the new version 1.

Reply:
fixed in: https://github.com/questdb/questdb/pull/5507/commits/18dd32480f52eef2f3aac2b537db27d64cbf1cee